Dockerfile

```
@@ -1,38 +1,52 @@
-ARG VERSION=unspecified
+ARG GIT_COMMIT=unspecified
+ARG GIT_REMOTE=unspecified
+ARG VERSION=0.0.1
 
-FROM python:3.9.6-alpine
+FROM python:3.9.6
 
+ARG GIT_COMMIT
+ARG GIT_REMOTE
 ARG VERSION
 
 # For a list of pre-defined annotation keys and value types see:
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 # Note: Additional labels are added by the build workflow.
+LABEL git_commit=$GIT_COMMIT
+LABEL git_remote=$GIT_REMOTE
 LABEL org.opencontainers.image.authors="mark.feldhousen@cisa.dhs.gov"
 LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
+LABEL version=$VERSION
 
 ARG CISA_UID=421
 ENV CISA_HOME="/home/cisa"
+ENV PCA_REPORT_TOOLS_SRC="/usr/src/pca-report-tools"
 ENV ECHO_MESSAGE="Hello World from Dockerfile"
 
 RUN addgroup --system --gid ${CISA_UID} cisa \
   && adduser --system --uid ${CISA_UID} --ingroup cisa cisa
 
-RUN apk --update --no-cache add \
-ca-certificates \
-openssl \
-py-pip
+RUN apt-get install wget
 
-WORKDIR ${CISA_HOME}
+RUN apt-get update && \
+ apt-get install --no-install-recommends -y texlive texlive-xetex texlive-bibtex-extra
 
-RUN wget -O sourcecode.tgz https://github.com/cisagov/skeleton-python-library/archive/v${VERSION}.tar.gz && \
+VOLUME $CISA_HOME
+
+COPY src/version.txt /src
+
+WORKDIR $PCA_REPORT_TOOLS_SRC
```

Use curly braces consistently for env variables:
```suggestion
WORKDIR ${PCA_REPORT_TOOLS_SRC}
```